images/zenko-zookeeper/kubernetes-entrypoint.sh

```diff
@@ -2,7 +2,8 @@
 
 set -eu
 
-ZOO_HEAP_SIZE=${ZOO_HEAP_SIZE:-2G}
+# Changes from kube to java friendly format 
+ZOO_HEAP_SIZE=`echo ${ZOO_HEAP_SIZE:-2G} | sed 's/Ti/T/; s/Gi/G/; s/Mi/M/'`
```

Comment:
@NicolasT How about something like this? The reason I refrained from using much regex before was to keep it as easily legible as possible but I think this still accomplishes that.